

<!DOCTYPE html>
<html class="writer-html4" lang="en" >
<head>
  <meta charset="utf-8" />
  
  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
  
  <title>Appendix &mdash; FuSViz 0.1.0 documentation</title>
  

  
  <link rel="stylesheet" href="_static/css/theme.css" type="text/css" />
  <link rel="stylesheet" href="_static/pygments.css" type="text/css" />
  <link rel="stylesheet" href="_static/css/custom.css" type="text/css" />

  
  

  
  

  

  
  <!--[if lt IE 9]>
    <script src="_static/js/html5shiv.min.js"></script>
  <![endif]-->
  
    
      <script type="text/javascript" id="documentation_options" data-url_root="./" src="_static/documentation_options.js"></script>
        <script type="text/javascript" src="_static/jquery.js"></script>
        <script type="text/javascript" src="_static/underscore.js"></script>
        <script type="text/javascript" src="_static/doctools.js"></script>
        <script type="text/javascript" src="_static/language_data.js"></script>
    
    <script type="text/javascript" src="_static/js/theme.js"></script>

    
    <link rel="index" title="Index" href="genindex.html" />
    <link rel="search" title="Search" href="search.html" />
    <link rel="next" title="Release_log" href="Release_log.html" />
    <link rel="prev" title="FAQ" href="FAQ.html" /> 
</head>

<body class="wy-body-for-nav">

   
  <div class="wy-grid-for-nav">
    
    <nav data-toggle="wy-nav-shift" class="wy-nav-side">
      <div class="wy-side-scroll">
        <div class="wy-side-nav-search" >
          

          
            <a href="index.html" class="icon icon-home"> FuSViz
          

          
          </a>

          
            
            
              <div class="version">
                0.1.0
              </div>
            
          

          
<div role="search">
  <form id="rtd-search-form" class="wy-form" action="search.html" method="get">
    <input type="text" name="q" placeholder="Search docs" />
    <input type="hidden" name="check_keywords" value="yes" />
    <input type="hidden" name="area" value="default" />
  </form>
</div>

          
        </div>

        
        <div class="wy-menu wy-menu-vertical" data-spy="affix" role="navigation" aria-label="main navigation">
          
            
            
              
            
            
              <p class="caption"><span class="caption-text">Table of Contents</span></p>
<ul class="current">
<li class="toctree-l1"><a class="reference internal" href="Introduction.html">Introduction</a></li>
<li class="toctree-l1"><a class="reference internal" href="Installation.html">Installation</a></li>
<li class="toctree-l1"><a class="reference internal" href="Input.html">Input</a></li>
<li class="toctree-l1"><a class="reference internal" href="Usage_manual.html">Usage manual</a></li>
<li class="toctree-l1"><a class="reference internal" href="FAQ.html">FAQ</a></li>
<li class="toctree-l1 current"><a class="current reference internal" href="#">Appendix</a><ul>
<li class="toctree-l2"><a class="reference internal" href="#load-local-alignment-file-in-linear-module">Load local alignment file in Linear module</a></li>
<li class="toctree-l2"><a class="reference internal" href="#load-alignment-track-from-url-address-in-linear-module">Load alignment track from URL address in Linear module</a></li>
<li class="toctree-l2"><a class="reference internal" href="#visualize-sv-event-together-with-read-coverage-using-two-way-module">Visualize SV event together with read coverage using Two-way module</a><ul>
<li class="toctree-l3"><a class="reference internal" href="#use-plot-separate-individual-bam-to-visualize-fusion-together-with-rna-seq-read-coverage">Use ‘plot_separate_individual_bam’ to visualize fusion together with RNA-seq read coverage</a></li>
<li class="toctree-l3"><a class="reference internal" href="#visualize-fusion-with-rna-seq-read-coverage-calculated-by-specific-transcript-isoforms">Visualize fusion with RNA-seq read coverage calculated by specific transcript isoforms</a></li>
<li class="toctree-l3"><a class="reference internal" href="#visualize-fusion-with-both-rna-seq-and-dna-seq-read-coverage">Visualize fusion with both RNA-seq and DNA-seq read coverage</a></li>
<li class="toctree-l3"><a class="reference internal" href="#a-full-usage-of-plot-separate-individual-bam">A full usage of ‘plot_separate_individual_bam’</a></li>
</ul>
</li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="Release_log.html">Release_log</a></li>
</ul>

            
          
        </div>
        
      </div>
    </nav>

    <section data-toggle="wy-nav-shift" class="wy-nav-content-wrap">

      
      <nav class="wy-nav-top" aria-label="top navigation">
        
          <i data-toggle="wy-nav-top" class="fa fa-bars"></i>
          <a href="index.html">FuSViz</a>
        
      </nav>


      <div class="wy-nav-content">
        
        <div class="rst-content">
        
          

















<div role="navigation" aria-label="breadcrumbs navigation">

  <ul class="wy-breadcrumbs">
    
      <li><a href="index.html" class="icon icon-home"></a> &raquo;</li>
        
      <li>Appendix</li>
    
    
      <li class="wy-breadcrumbs-aside">
        
          
            <a href="_sources/Appendix.rst.txt" rel="nofollow"> View page source</a>
          
        
      </li>
    
  </ul>

  
  <hr/>
</div>
          <div role="main" class="document" itemscope="itemscope" itemtype="http://schema.org/Article">
           <div itemprop="articleBody">
            
  <div class="section" id="appendix">
<h1>Appendix<a class="headerlink" href="#appendix" title="Permalink to this headline">¶</a></h1>
<p>Although FuSViz is designed for SV interpretation and visualization of
multiple samples, it could be utilized for single sample analysis
together with read alignments as well. Currently, import read alignment
files is allowed in <strong>Linear</strong> and <strong>Two-way</strong> modules.</p>
<div class="section" id="load-local-alignment-file-in-linear-module">
<h2>Load local alignment file in Linear module<a class="headerlink" href="#load-local-alignment-file-in-linear-module" title="Permalink to this headline">¶</a></h2>
<p>Alignment and index files need to be uploaded together. For example,
import an RNA-seq alignment in BAM format.</p>
<div class="figure">
<img alt="" src="_images/6.1.Load_local_alignment_track_in_Linear_module_1.png" />
</div>
<div class="figure">
<img alt="" src="_images/6.1.Load_local_alignment_track_in_Linear_module_2.png" />
</div>
<p>Here, a <strong>split-window</strong> mode is used to investigate alignment quality
of split reads or discordant read pairs mapped to different genomic
loci. The read pair highlighted in the dash boxes shows a discordant
mapping feature to partner genes ABHD12B and RCC1, respectively.</p>
</div>
<div class="section" id="load-alignment-track-from-url-address-in-linear-module">
<h2>Load alignment track from URL address in Linear module<a class="headerlink" href="#load-alignment-track-from-url-address-in-linear-module" title="Permalink to this headline">¶</a></h2>
<p>If alignment data is shared via cloud or hosted in a remote server,
users can load it together with index via URL web address. For example,</p>
<div class="figure">
<img alt="" src="_images/6.2.Load_alignment_track_from_URL_address.png" />
</div>
</div>
<div class="section" id="visualize-sv-event-together-with-read-coverage-using-two-way-module">
<h2>Visualize SV event together with read coverage using Two-way module<a class="headerlink" href="#visualize-sv-event-together-with-read-coverage-using-two-way-module" title="Permalink to this headline">¶</a></h2>
<p>It has to be done under command line (<strong>NOT</strong> available in web
interface) using the function <code class="docutils literal notranslate"><span class="pre">plot_separate_individual_bam</span></code>. Firstly,
load FuSViz package and genomic annotation object as below:</p>
<div class="highlight-default notranslate"><div class="highlight"><pre><span></span><span class="n">library</span><span class="p">(</span><span class="n">FuSViz</span><span class="p">)</span>
<span class="n">options</span><span class="p">(</span><span class="n">uscsChromosomeName</span><span class="o">=</span><span class="n">FALSE</span><span class="p">)</span>
</pre></div>
</div>
<p>Import gene symbol and synonymous names</p>
<div class="highlight-default notranslate"><div class="highlight"><pre><span></span>system_path = path.package(&quot;FuSViz&quot;)
extdata = system.file(&quot;extdata&quot;, package = &quot;FuSViz&quot;)

load(file=file.path(extdata, &quot;ensembl_symbol.Rd&quot;))
symbol_ensem = gene_id$Gene_name
names(symbol_ensem) = gene_id$Gene_ID
</pre></div>
</div>
<p>Set gene/transcript annotation version (e.g. <strong>hg19</strong>; it needs to
change to <strong>hg38</strong> if genomic coordinates of breakpoints come from human
genome reference <strong>hg38</strong>).</p>
<div class="highlight-default notranslate"><div class="highlight"><pre><span></span><span class="n">version</span> <span class="o">=</span> <span class="s2">&quot;hg19&quot;</span><span class="p">;</span> <span class="c1"># or “hg38”</span>
</pre></div>
</div>
<p>Load gene/transcript annotation object:</p>
<div class="highlight-default notranslate"><div class="highlight"><pre><span></span><span class="n">txdb</span> <span class="o">=</span> <span class="n">suppressWarnings</span><span class="p">(</span><span class="n">suppressPackageStartupMessages</span><span class="p">(</span><span class="n">AnnotationDbi</span><span class="p">::</span><span class="n">loadDb</span><span class="p">(</span><span class="n">file</span><span class="o">=</span><span class="n">file</span><span class="o">.</span><span class="n">path</span><span class="p">(</span><span class="n">extdata</span><span class="p">,</span> <span class="n">paste</span><span class="p">(</span><span class="s2">&quot;gencode.v36.annotation.&quot;</span><span class="p">,</span> <span class="n">version</span><span class="p">,</span> <span class="s2">&quot;.sqlite&quot;</span><span class="p">,</span> <span class="n">sep</span><span class="o">=</span><span class="s2">&quot;&quot;</span><span class="p">)))))</span>
<span class="n">load</span><span class="p">(</span><span class="n">file</span><span class="o">=</span><span class="n">file</span><span class="o">.</span><span class="n">path</span><span class="p">(</span><span class="n">extdata</span><span class="p">,</span> <span class="n">paste</span><span class="p">(</span><span class="s2">&quot;grTrack.&quot;</span><span class="p">,</span> <span class="n">version</span><span class="p">,</span> <span class="s2">&quot;.Rd&quot;</span><span class="p">,</span> <span class="n">sep</span><span class="o">=</span><span class="s2">&quot;&quot;</span><span class="p">)))</span>
<span class="n">load</span><span class="p">(</span><span class="n">file</span><span class="o">=</span><span class="n">file</span><span class="o">.</span><span class="n">path</span><span class="p">(</span><span class="n">extdata</span><span class="p">,</span> <span class="n">paste</span><span class="p">(</span><span class="s2">&quot;cytoband.&quot;</span><span class="p">,</span> <span class="n">version</span><span class="p">,</span> <span class="s2">&quot;.Rd&quot;</span><span class="p">,</span> <span class="n">sep</span><span class="o">=</span><span class="s2">&quot;&quot;</span><span class="p">)))</span>
<span class="n">whole_txdb</span> <span class="o">=</span> <span class="n">GenomicFeatures</span><span class="p">::</span><span class="n">exonsBy</span><span class="p">(</span><span class="n">txdb</span><span class="p">,</span> <span class="n">by</span> <span class="o">=</span> <span class="s2">&quot;tx&quot;</span><span class="p">,</span> <span class="n">use</span><span class="o">.</span><span class="n">names</span><span class="o">=</span><span class="n">TRUE</span><span class="p">);</span> <span class="c1"># group exons by transcript_id</span>
</pre></div>
</div>
<div class="section" id="use-plot-separate-individual-bam-to-visualize-fusion-together-with-rna-seq-read-coverage">
<h3>Use ‘plot_separate_individual_bam’ to visualize fusion together with RNA-seq read coverage<a class="headerlink" href="#use-plot-separate-individual-bam-to-visualize-fusion-together-with-rna-seq-read-coverage" title="Permalink to this headline">¶</a></h3>
<p>For example, plot a fusion event of “PIK3R1-HSD17B4”</p>
<div class="highlight-default notranslate"><div class="highlight"><pre><span></span><span class="n">rna_bam_path</span><span class="o">=</span><span class="n">file</span><span class="o">.</span><span class="n">path</span><span class="p">(</span><span class="n">extdata</span><span class="p">,</span> <span class="s2">&quot;RNA-seq-example.bam&quot;</span><span class="p">)</span>
<span class="n">plot_separate_individual_bam</span><span class="p">(</span><span class="n">first_name</span> <span class="o">=</span> <span class="s2">&quot;PIK3R1&quot;</span><span class="p">,</span> <span class="n">second_name</span> <span class="o">=</span> <span class="s2">&quot;HSD17B4&quot;</span><span class="p">,</span> <span class="n">breakpoint_A</span> <span class="o">=</span> <span class="mi">67576834</span><span class="p">,</span> <span class="n">breakpoint_B</span> <span class="o">=</span> <span class="mi">118792010</span><span class="p">,</span> <span class="n">coverage_plot_trans</span> <span class="o">=</span> <span class="n">F</span><span class="p">,</span> <span class="n">rna_bam_path</span> <span class="o">=</span> <span class="n">rna_bam_path</span><span class="p">,</span> <span class="n">split</span> <span class="o">=</span> <span class="mi">9</span><span class="p">,</span> <span class="n">span</span> <span class="o">=</span> <span class="mi">1</span><span class="p">,</span> <span class="n">fusion_strandA</span><span class="o">=</span><span class="s2">&quot;+&quot;</span><span class="p">,</span> <span class="n">fusion_strandB</span><span class="o">=</span><span class="s2">&quot;-&quot;</span><span class="p">)</span>
</pre></div>
</div>
<div class="figure">
<img alt="" src="_images/6.3.fusion_RNA-seq_1.png" />
</div>
<p>From the top it shows the position of partner genes in a chromosome
ideogram, the fusion event (a curved line marked by read support (9 -
split read, 1 – spanning read pair); arrow indicates transcribed
direction of the fusion), exon annotations of different transcript
isoforms for upstream (colored by <strong>green</strong>) and downstream (colored by
<strong>orange</strong>) partners, RNA expression levels measured by read counts and
genomic coordinates of partner gene loci in Mb from chromosome.</p>
</div>
<div class="section" id="visualize-fusion-with-rna-seq-read-coverage-calculated-by-specific-transcript-isoforms">
<h3>Visualize fusion with RNA-seq read coverage calculated by specific transcript isoforms<a class="headerlink" href="#visualize-fusion-with-rna-seq-read-coverage-calculated-by-specific-transcript-isoforms" title="Permalink to this headline">¶</a></h3>
<div class="highlight-default notranslate"><div class="highlight"><pre><span></span><span class="n">plot_separate_individual_bam</span><span class="p">(</span><span class="n">first_name</span> <span class="o">=</span> <span class="s2">&quot;PIK3R1&quot;</span><span class="p">,</span> <span class="n">second_name</span> <span class="o">=</span> <span class="s2">&quot;HSD17B4&quot;</span><span class="p">,</span> <span class="n">breakpoint_A</span> <span class="o">=</span> <span class="mi">67576834</span><span class="p">,</span> <span class="n">breakpoint_B</span> <span class="o">=</span> <span class="mi">118792010</span><span class="p">,</span> <span class="n">coverage_plot_trans</span> <span class="o">=</span> <span class="n">T</span><span class="p">,</span> <span class="n">rna_bam_path</span> <span class="o">=</span> <span class="n">rna_bam_path</span><span class="p">,</span> <span class="n">transcriptA</span><span class="o">=</span><span class="s2">&quot;ENST00000521381&quot;</span><span class="p">,</span> <span class="n">transcriptB</span><span class="o">=</span><span class="s2">&quot;ENST00000414835&quot;</span><span class="p">,</span> <span class="n">split</span> <span class="o">=</span> <span class="mi">9</span><span class="p">,</span> <span class="n">span</span> <span class="o">=</span> <span class="mi">1</span><span class="p">,</span> <span class="n">fusion_strandA</span><span class="o">=</span><span class="s2">&quot;+&quot;</span><span class="p">,</span> <span class="n">fusion_strandB</span><span class="o">=</span><span class="s2">&quot;-&quot;</span><span class="p">)</span>
</pre></div>
</div>
<p><code class="docutils literal notranslate"><span class="pre">coverage_plot_trans</span> <span class="pre">=</span> <span class="pre">T</span></code> suggests RNA-seq read coverage is measured
using the chosen transcripts <strong>ENST00000521381</strong> and
<strong>ENST00000414835</strong>.</p>
<div class="figure">
<img alt="" src="_images/6.3.fusion_RNA-seq_2.png" />
</div>
</div>
<div class="section" id="visualize-fusion-with-both-rna-seq-and-dna-seq-read-coverage">
<h3>Visualize fusion with both RNA-seq and DNA-seq read coverage<a class="headerlink" href="#visualize-fusion-with-both-rna-seq-and-dna-seq-read-coverage" title="Permalink to this headline">¶</a></h3>
<div class="highlight-default notranslate"><div class="highlight"><pre><span></span><span class="n">dna_bam_path</span><span class="o">=</span><span class="n">file</span><span class="o">.</span><span class="n">path</span><span class="p">(</span><span class="n">extdata</span><span class="p">,</span> <span class="s2">&quot;DNA-seq-example.bam&quot;</span><span class="p">);</span>
<span class="n">plot_separate_individual_bam</span><span class="p">(</span><span class="n">first_name</span> <span class="o">=</span> <span class="s2">&quot;PIK3R1&quot;</span><span class="p">,</span> <span class="n">second_name</span> <span class="o">=</span> <span class="s2">&quot;HSD17B4&quot;</span><span class="p">,</span> <span class="n">breakpoint_A</span> <span class="o">=</span> <span class="mi">67576834</span><span class="p">,</span> <span class="n">breakpoint_B</span> <span class="o">=</span> <span class="mi">118792010</span><span class="p">,</span> <span class="n">coverage_plot_trans</span> <span class="o">=</span> <span class="n">F</span><span class="p">,</span> <span class="n">chrom_notation_rna</span> <span class="o">=</span> <span class="n">T</span><span class="p">,</span> <span class="n">chrom_notation_dna</span> <span class="o">=</span> <span class="n">F</span><span class="p">,</span> <span class="n">split</span> <span class="o">=</span> <span class="mi">9</span><span class="p">,</span> <span class="n">span</span> <span class="o">=</span> <span class="mi">1</span><span class="p">,</span> <span class="n">rna_bam_path</span> <span class="o">=</span> <span class="n">rna_bam_path</span><span class="p">,</span> <span class="n">dna_bam_path</span> <span class="o">=</span> <span class="n">dna_bam_path</span><span class="p">,</span> <span class="n">fusion_strandA</span><span class="o">=</span><span class="s2">&quot;+&quot;</span><span class="p">,</span> <span class="n">fusion_strandB</span><span class="o">=</span><span class="s2">&quot;-&quot;</span><span class="p">)</span>
</pre></div>
</div>
<p>Read alignment BAM file from DNA-seq can be whole genome sequencing,
Exome-seq or gene-panel target sequencing. <code class="docutils literal notranslate"><span class="pre">chrom_notation_rna</span> <span class="pre">=</span> <span class="pre">T</span></code>
suggests the chromosome notation in RNA-seq alignment file is named like
<strong>‘chrX’</strong> (i.e. UCSC syntax); <code class="docutils literal notranslate"><span class="pre">chrom_notation_dna</span> <span class="pre">=</span> <span class="pre">F</span></code> denotes the
chromosome notation in DNA-seq alignment file is named like <strong>‘X’</strong>
(i.e. ensembl syntax).</p>
<div class="figure">
<img alt="" src="_images/6.3.fusion_RNA-seq_DNA-seq_3.png" />
</div>
</div>
<div class="section" id="a-full-usage-of-plot-separate-individual-bam">
<h3>A full usage of ‘plot_separate_individual_bam’<a class="headerlink" href="#a-full-usage-of-plot-separate-individual-bam" title="Permalink to this headline">¶</a></h3>
<p>See reference, <code class="docutils literal notranslate"><span class="pre">?plot_separate_individual_bam</span></code></p>
</div>
</div>
</div>


           </div>
           
          </div>
          <footer>
    <div class="rst-footer-buttons" role="navigation" aria-label="footer navigation">
        <a href="Release_log.html" class="btn btn-neutral float-right" title="Release_log" accesskey="n" rel="next">Next <span class="fa fa-arrow-circle-right" aria-hidden="true"></span></a>
        <a href="FAQ.html" class="btn btn-neutral float-left" title="FAQ" accesskey="p" rel="prev"><span class="fa fa-arrow-circle-left" aria-hidden="true"></span> Previous</a>
    </div>

  <hr/>

  <div role="contentinfo">
    <p>
        &#169; Copyright 2021, Sen ZHAO.

    </p>
  </div>
    
    
    
    Built with <a href="https://www.sphinx-doc.org/">Sphinx</a> using a
    
    <a href="https://github.com/readthedocs/sphinx_rtd_theme">theme</a>
    
    provided by <a href="https://readthedocs.org">Read the Docs</a>. 

</footer>
        </div>
      </div>

    </section>

  </div>
  

  <script type="text/javascript">
      jQuery(function () {
          SphinxRtdTheme.Navigation.enable(true);
      });
  </script>

  
  
    
   

</body>
</html>