

<!DOCTYPE html>
<html class="writer-html4" lang="en" >
<head>
  <meta charset="utf-8" />
  
  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
  
  <title>FAQ &mdash; FuSViz 0.1.0 documentation</title>
  

  
  <link rel="stylesheet" href="_static/css/theme.css" type="text/css" />
  <link rel="stylesheet" href="_static/pygments.css" type="text/css" />
  <link rel="stylesheet" href="_static/css/custom.css" type="text/css" />

  
  

  
  

  

  
  <!--[if lt IE 9]>
    <script src="_static/js/html5shiv.min.js"></script>
  <![endif]-->
  
    
      <script type="text/javascript" id="documentation_options" data-url_root="./" src="_static/documentation_options.js"></script>
        <script type="text/javascript" src="_static/jquery.js"></script>
        <script type="text/javascript" src="_static/underscore.js"></script>
        <script type="text/javascript" src="_static/doctools.js"></script>
        <script type="text/javascript" src="_static/language_data.js"></script>
    
    <script type="text/javascript" src="_static/js/theme.js"></script>

    
    <link rel="index" title="Index" href="genindex.html" />
    <link rel="search" title="Search" href="search.html" />
    <link rel="next" title="Appendix" href="Appendix.html" />
    <link rel="prev" title="Usage manual" href="Usage_manual.html" /> 
</head>

<body class="wy-body-for-nav">

   
  <div class="wy-grid-for-nav">
    
    <nav data-toggle="wy-nav-shift" class="wy-nav-side">
      <div class="wy-side-scroll">
        <div class="wy-side-nav-search" >
          

          
            <a href="index.html" class="icon icon-home"> FuSViz
          

          
          </a>

          
            
            
              <div class="version">
                0.1.0
              </div>
            
          

          
<div role="search">
  <form id="rtd-search-form" class="wy-form" action="search.html" method="get">
    <input type="text" name="q" placeholder="Search docs" />
    <input type="hidden" name="check_keywords" value="yes" />
    <input type="hidden" name="area" value="default" />
  </form>
</div>

          
        </div>

        
        <div class="wy-menu wy-menu-vertical" data-spy="affix" role="navigation" aria-label="main navigation">
          
            
            
              
            
            
              <p class="caption"><span class="caption-text">Table of Contents</span></p>
<ul class="current">
<li class="toctree-l1"><a class="reference internal" href="Introduction.html">Introduction</a></li>
<li class="toctree-l1"><a class="reference internal" href="Installation.html">Installation</a></li>
<li class="toctree-l1"><a class="reference internal" href="Input.html">Input</a></li>
<li class="toctree-l1"><a class="reference internal" href="Usage_manual.html">Usage manual</a></li>
<li class="toctree-l1 current"><a class="current reference internal" href="#">FAQ</a></li>
<li class="toctree-l1"><a class="reference internal" href="Appendix.html">Appendix</a></li>
<li class="toctree-l1"><a class="reference internal" href="Release_log.html">Release_log</a></li>
</ul>

            
          
        </div>
        
      </div>
    </nav>

    <section data-toggle="wy-nav-shift" class="wy-nav-content-wrap">

      
      <nav class="wy-nav-top" aria-label="top navigation">
        
          <i data-toggle="wy-nav-top" class="fa fa-bars"></i>
          <a href="index.html">FuSViz</a>
        
      </nav>


      <div class="wy-nav-content">
        
        <div class="rst-content">
        
          

















<div role="navigation" aria-label="breadcrumbs navigation">

  <ul class="wy-breadcrumbs">
    
      <li><a href="index.html" class="icon icon-home"></a> &raquo;</li>
        
      <li>FAQ</li>
    
    
      <li class="wy-breadcrumbs-aside">
        
          
            <a href="_sources/FAQ.rst.txt" rel="nofollow"> View page source</a>
          
        
      </li>
    
  </ul>

  
  <hr/>
</div>
          <div role="main" class="document" itemscope="itemscope" itemtype="http://schema.org/Article">
           <div itemprop="articleBody">
            
  <div class="section" id="faq">
<h1>FAQ<a class="headerlink" href="#faq" title="Permalink to this headline">¶</a></h1>
<p>Frequently asked questions regarding to installation and usage of
<strong>FuSViz</strong>.</p>
<p><strong>1. How to prepare input format for FuSViz?</strong></p>
<ul class="simple">
<li>FuSViz accepts three types of data as input: SV callings from
RNA-seq/DNA-seq generated using illumina platform and mutation
profile (i.e. small variants). Here, an in-house script
<strong>SV_standard</strong> (see its usage in <strong>Installation</strong> section) is
provided to convert raw SV callings from various finders to a uniform
format, then aggregate callings of multiple samples together. In
terms of SVs from RNA-seq data, numerous callers have been developed
and there lacks of a standardization format for the output. In the
processing of <strong>SV_standard</strong>, a few common features (e.g. gene
name, breakpoint position, sample id, read support for SV event)
present in output of most callers are extracted and standardized as
an input format of FuSViz. In contrast, most of DNA-seq SV finders
output callings in a standard VCF format, which is converted to bedpe
format (a “simplified” version keeps some key fields in VCF file that
are informative for FuSViz analysis) by <strong>SV_standard</strong>. Currently,
output format of <strong>YY</strong> SV callers (STAR-fusion, Arriba, deFuse,
FusionCatcher, SOAPFuse and Pizzly for RNA-seq; Menta, GRISSS, Lumpy,
Delly and SvABA for DNA-seq) are well supported by <strong>SV_standard</strong>,
and we will of course extend to more callers on basis of user’s
requirement and feedback in the further.</li>
</ul>
<p><strong>2. What are differences between SV callings from DNA-seq and RNA-seq,
and their respective characteristics? Why to combine them?</strong></p>
<ul class="simple">
<li>Genomic SV at DNA level represents various types of large variations
(e.g. deletions, insertions, duplications, inversions and
translocations) that are related to the break, link, recombination
and amplification of DNA strand. If it occurs in genic regions, a
hybrid gene could be formed (so-called “fusion gene”). In case of
breakpoint in intergenic regions, duplications and deletions can lead
to an alteration of genomic regulatory elements (e.g. enhancers or
TADs). One common phenomenon of SV rearrangement is to break up DNA
sequence, which disrupts proper expression or translation of a gene
(e.g. tumor suppressor), finally giving rise to a reduced activation
or function loss of such gene.</li>
<li>In comparison, RNA-seq data is used to address whether a fusion gene
could transcribe to a fusion transcript with an in-frame ORF and
produce a putative chimeric protein. Although breakpoints of fusion
genes usually occur in intron regions, the observed breakpoints of
most fusion transcripts coincide with exon boundary due to RNA
splicing mechanism, from which some breakpoint pairs show a higher
prevalence than others. One important biological consequence of
fusion transcripts is an activation of oncogenes, as the oncogenes
often constitutes the downstream partner in fusion. Even when the
oncogene is fused as upstream partner, the downstream partner can
provide protein domains which enhance oncogenic function if read
frame is intact. In addition, a few SV types, e.g. <strong>read-through</strong>
and <strong>trans-splicing</strong>, are exclusively detected at RNA level because
they are generated by RNA-mediated regulatory mechanism. A common
limitation of RNA-seq data analysis is that the investigation of
intergenic SV events is not available.</li>
<li>In short, the features of SV callings from DNA-seq and RNA-seq show
complementary with each other. Thus, it is reasonable to combine them
together to interpret and visualize SVs associated with
oncological-relevant genes in context of cancer genomic and
transcriptomic annotations.</li>
</ul>
<p><strong>3. Are SV callings from the third-generation sequencing technology
(e.g. PacBio or Nanopore platform) accepted by FuSViz.</strong></p>
<ul class="simple">
<li>Yes, the raw callings have to be processed by <strong>SV_standard</strong> before
feeding in FuSViz. As PacBio represents a long-read sequencing
technology, only “split” reads are used for SV detection and no
“span” reads information is available in the output of SV callers.
So, <strong>SV_standard</strong> will assign a “virtual” number to avoid any ‘NA’
or empty values in “span” column during processing.</li>
</ul>
<p><strong>4. If I am only interested in copy number aberrations (CNAs), how
could I make a filtration on SVs.</strong></p>
<ul class="simple">
<li>In general, CNAs show unusual amplifications and deletions of genomic
regions in a chromosome. They represent intra-chromosome events and
are in fact a subset of total SVs, which are tagged as <strong>DUP</strong> and
<strong>DEL</strong> in the input format of FuSViz. Such an analysis is provided
by clicking the button <code class="docutils literal notranslate"><span class="pre">DNA</span> <span class="pre">SV</span> <span class="pre">Track</span> <span class="pre">in</span> <span class="pre">seg</span></code> in Linear plot module.
In addition, it can be done by using <code class="docutils literal notranslate"><span class="pre">SV_type</span></code> Select DropDown to
control the type of SVs for customized analysis.</li>
</ul>
<p><strong>5. Can it be possible to utilize FuSViz for analysis of a single
sample and ‘tumor-normal’ pairwise samples?</strong></p>
<ul class="simple">
<li>Yes, there are two ways for it. First, SV callings or mutation
profile from a single sample are allowed for input. Second, user
could utilize <code class="docutils literal notranslate"><span class="pre">name</span></code> or <code class="docutils literal notranslate"><span class="pre">Sample</span></code> Select DropDown to customize
their interested samples. Moreover, a few functinalities are provided
to enhance the single sample analysis by allowing import of read
alignment file (see <strong>Appendix</strong> section). However, a visualization
strategy for “tumor-normal” pairwise samples is not supported for
FuSViz currently.</li>
</ul>
<p><strong>6. Is it possible for FuSViz to analyze SVs callings from Non-human
organisms?</strong></p>
<ul class="simple">
<li>Not yet now, but we are working on it and trying to extend FuSViz to
other model organisms (e.g. mouse and rat).</li>
</ul>
<p><strong>7. Do SV callings in FuSViz input file include genotype information?</strong></p>
<ul class="simple">
<li>No, there are several reasons for it:<ul>
<li>Firstly, only a few of existed callers (around 1/3) provide
genotype information of SVs in raw output.</li>
<li>Secondly, even one tool has a functionality for genotyping SVs,
and it actually depends on the type of SVs. In general,
duplication and deletion can be genotyped relatively accurate.
Sometimes, genotype of insertion and inversion can be inferred but
less precision. It is still challenging to genotype translocation
for any available tools.</li>
<li>Thirdly, it is often difficult to obtain a genotype information if
SV callings from multiple tools are merged, in particular when
there are incongruent among different callers. In addition, FuSViz
is mainly used for SV analysis of multiple samples and it is not
crucial to have a genotype information available for that.</li>
</ul>
</li>
</ul>
<p><strong>8. Does FuSViz have a functionality to allow alignment file (BAM or
CRAM) as input?</strong></p>
<ul class="simple">
<li>As the main purpose of FuSViz is used to visualize and interpret SVs
of multiple samples, it does not make sense to import read alignments
of multiple samples. But alignment file (either from RNA-seq or
DNA-seq or both) of one specific sample is allowed to load for a
customized single-sample invesgitation in <strong>Linear</strong> and <strong>Two-way</strong>
modules (see <strong>Appendix</strong>).</li>
</ul>
</div>


           </div>
           
          </div>
          <footer>
    <div class="rst-footer-buttons" role="navigation" aria-label="footer navigation">
        <a href="Appendix.html" class="btn btn-neutral float-right" title="Appendix" accesskey="n" rel="next">Next <span class="fa fa-arrow-circle-right" aria-hidden="true"></span></a>
        <a href="Usage_manual.html" class="btn btn-neutral float-left" title="Usage manual" accesskey="p" rel="prev"><span class="fa fa-arrow-circle-left" aria-hidden="true"></span> Previous</a>
    </div>

  <hr/>

  <div role="contentinfo">
    <p>
        &#169; Copyright 2021, Sen ZHAO.

    </p>
  </div>
    
    
    
    Built with <a href="https://www.sphinx-doc.org/">Sphinx</a> using a
    
    <a href="https://github.com/readthedocs/sphinx_rtd_theme">theme</a>
    
    provided by <a href="https://readthedocs.org">Read the Docs</a>. 

</footer>
        </div>
      </div>

    </section>

  </div>
  

  <script type="text/javascript">
      jQuery(function () {
          SphinxRtdTheme.Navigation.enable(true);
      });
  </script>

  
  
    
   

</body>
</html>