

<!DOCTYPE html>
<html class="writer-html4" lang="en" >
<head>
  <meta charset="utf-8" />
  
  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
  
  <title>Input &mdash; FuSViz 0.1.0 documentation</title>
  

  
  <link rel="stylesheet" href="_static/css/theme.css" type="text/css" />
  <link rel="stylesheet" href="_static/pygments.css" type="text/css" />
  <link rel="stylesheet" href="_static/css/custom.css" type="text/css" />

  
  

  
  

  

  
  <!--[if lt IE 9]>
    <script src="_static/js/html5shiv.min.js"></script>
  <![endif]-->
  
    
      <script type="text/javascript" id="documentation_options" data-url_root="./" src="_static/documentation_options.js"></script>
        <script type="text/javascript" src="_static/jquery.js"></script>
        <script type="text/javascript" src="_static/underscore.js"></script>
        <script type="text/javascript" src="_static/doctools.js"></script>
        <script type="text/javascript" src="_static/language_data.js"></script>
    
    <script type="text/javascript" src="_static/js/theme.js"></script>

    
    <link rel="index" title="Index" href="genindex.html" />
    <link rel="search" title="Search" href="search.html" />
    <link rel="next" title="Usage manual" href="Usage_manual.html" />
    <link rel="prev" title="Installation" href="Installation.html" /> 
</head>

<body class="wy-body-for-nav">

   
  <div class="wy-grid-for-nav">
    
    <nav data-toggle="wy-nav-shift" class="wy-nav-side">
      <div class="wy-side-scroll">
        <div class="wy-side-nav-search" >
          

          
            <a href="index.html" class="icon icon-home"> FuSViz
          

          
          </a>

          
            
            
              <div class="version">
                0.1.0
              </div>
            
          

          
<div role="search">
  <form id="rtd-search-form" class="wy-form" action="search.html" method="get">
    <input type="text" name="q" placeholder="Search docs" />
    <input type="hidden" name="check_keywords" value="yes" />
    <input type="hidden" name="area" value="default" />
  </form>
</div>

          
        </div>

        
        <div class="wy-menu wy-menu-vertical" data-spy="affix" role="navigation" aria-label="main navigation">
          
            
            
              
            
            
              <p class="caption"><span class="caption-text">Table of Contents</span></p>
<ul class="current">
<li class="toctree-l1"><a class="reference internal" href="Introduction.html">Introduction</a></li>
<li class="toctree-l1"><a class="reference internal" href="Installation.html">Installation</a></li>
<li class="toctree-l1 current"><a class="current reference internal" href="#">Input</a><ul>
<li class="toctree-l2"><a class="reference internal" href="#input-of-sv-callings-from-dna-seq-rna-seq-data">Input of SV callings from DNA-seq / RNA-seq data</a><ul>
<li class="toctree-l3"><a class="reference internal" href="#use-sv-standard-to-merge-aggregate-sv-callings-from-dna-seq-data">Use <strong>SV_standard</strong> to merge/aggregate SV callings from DNA-seq data</a></li>
<li class="toctree-l3"><a class="reference internal" href="#use-sv-standard-to-merge-aggregate-sv-callings-from-rna-seq-data">Use <strong>SV_standard</strong> to merge/aggregate SV callings from RNA-seq data</a></li>
<li class="toctree-l3"><a class="reference internal" href="#for-usage-of-sv-standard-arguments">For usage of <strong>SV_standard</strong> arguments</a></li>
<li class="toctree-l3"><a class="reference internal" href="#run-sv-standard-via-singularity-container">Run <strong>SV_standard</strong> via Singularity container</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="#input-of-mutation-profile-optional">Input of mutation profile (optional)</a></li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="Usage_manual.html">Usage manual</a></li>
<li class="toctree-l1"><a class="reference internal" href="FAQ.html">FAQ</a></li>
<li class="toctree-l1"><a class="reference internal" href="Appendix.html">Appendix</a></li>
<li class="toctree-l1"><a class="reference internal" href="Release_log.html">Release_log</a></li>
</ul>

            
          
        </div>
        
      </div>
    </nav>

    <section data-toggle="wy-nav-shift" class="wy-nav-content-wrap">

      
      <nav class="wy-nav-top" aria-label="top navigation">
        
          <i data-toggle="wy-nav-top" class="fa fa-bars"></i>
          <a href="index.html">FuSViz</a>
        
      </nav>


      <div class="wy-nav-content">
        
        <div class="rst-content">
        
          

















<div role="navigation" aria-label="breadcrumbs navigation">

  <ul class="wy-breadcrumbs">
    
      <li><a href="index.html" class="icon icon-home"></a> &raquo;</li>
        
      <li>Input</li>
    
    
      <li class="wy-breadcrumbs-aside">
        
          
            <a href="_sources/Input.rst.txt" rel="nofollow"> View page source</a>
          
        
      </li>
    
  </ul>

  
  <hr/>
</div>
          <div role="main" class="document" itemscope="itemscope" itemtype="http://schema.org/Article">
           <div itemprop="articleBody">
            
  <div class="section" id="input">
<h1>Input<a class="headerlink" href="#input" title="Permalink to this headline">¶</a></h1>
<p>FuSViz accepts input files from three sources:</p>
<ul class="simple">
<li>Aggregated SV callings from multiple samples RNA-seq data (e.g.
<a class="reference external" href="https://fusviz.s3.eu-north-1.amazonaws.com/RNA_SV_example.txt">RNA_SV_example.txt</a>)</li>
<li>Aggregated SV callings from multiple samples DNA-seq data (e.g.
<a class="reference external" href="https://fusviz.s3.eu-north-1.amazonaws.com/DNA_SV_example.txt">DNA_SV_example.txt</a>)</li>
<li>Aggregated mutation variants (SNVs+Indels) from multiple samples
DNA-seq data (e.g.
<a class="reference external" href="https://fusviz.s3.eu-north-1.amazonaws.com/TCGA.PRAD.mutect.somatic.maf">TCGA.PRAD.mutect.somatic.maf</a>)</li>
</ul>
<p>To run FuSViz needs SV callings from either one or both of DNA-seq and
RNA-seq. Mutation profile is an optional, but recommended.</p>
<div class="section" id="input-of-sv-callings-from-dna-seq-rna-seq-data">
<h2>Input of SV callings from DNA-seq / RNA-seq data<a class="headerlink" href="#input-of-sv-callings-from-dna-seq-rna-seq-data" title="Permalink to this headline">¶</a></h2>
<p>FuSViz does not take raw output of any SV callers as input. Here, we
provide an in-house script <strong>SV_standard</strong> to prepare the input format
for FuSViz. In short, SV callings of all samples are aggregated directly
if only one caller is used for detection. Sometimes multiple callers are
utilized - <strong>SV_standard</strong> firstly merges and makes a consensus set
from output of multiple callers for one sample, then aggregates the
results of all samples. This script depends on the tool
<a class="reference external" href="https://github.com/fritzsedlazeck/SURVIVOR">SURVIVOR</a> (download and
install it according to the developers’ instructions and make it
available in your <code class="docutils literal notranslate"><span class="pre">$PATH</span></code>).</p>
<div class="section" id="use-sv-standard-to-merge-aggregate-sv-callings-from-dna-seq-data">
<h3>Use <strong>SV_standard</strong> to merge/aggregate SV callings from DNA-seq data<a class="headerlink" href="#use-sv-standard-to-merge-aggregate-sv-callings-from-dna-seq-data" title="Permalink to this headline">¶</a></h3>
<p>Run an example:</p>
<div class="highlight-default notranslate"><div class="highlight"><pre><span></span>./SV_standard \
--type DNA \
--genome hg38 \
--Manta “/path/folder_Manta” \ #// contains multiple-sample DNA SV callings in VCF format from Manta
--Lumpy &quot;/path/folder_Lumpy&quot; \ #// contains multiple-sample DNA SV callings in VCF format from Lumpy
--Delly &quot;/path/folder_Delly&quot; \ #// contains multiple-sample DNA SV callings in VCF format from Delly
--output &quot;“/path/ouput_dna.bedpe&quot;
</pre></div>
</div>
<ul class="simple">
<li>VCF file should be uncompressed.</li>
<li>VCF file name of each sample in “/path/folder_Manta”,
“/path/folder_Delly” and “/path/folder_Lumpy” should be identical.</li>
<li>SVs tagged as “PASS” and “.” in VCF file are kept in final result.</li>
<li>Currently, output from SV callers (Menta, GRISSS, Lumpy, Delly and
SvABA) is supported by <strong>SV_standard</strong>.</li>
</ul>
</div>
<div class="section" id="use-sv-standard-to-merge-aggregate-sv-callings-from-rna-seq-data">
<h3>Use <strong>SV_standard</strong> to merge/aggregate SV callings from RNA-seq data<a class="headerlink" href="#use-sv-standard-to-merge-aggregate-sv-callings-from-rna-seq-data" title="Permalink to this headline">¶</a></h3>
<p>Run an example:</p>
<div class="highlight-default notranslate"><div class="highlight"><pre><span></span>./SV_standard \
--type RNA \
--genome hg38 \
--Arriba “/path/folder_Arriba” \ #// contains multiple-sample RNA SV callings from Arriba
--STAR-Fusion “/path/folder_STAR-Fusion” \ #// contains multiple-sample RNA SV callings from STAR-Fusion
--FusionCatcher “/path/folder_FusionCatcher” \ #// contains multiple-sample RNA SV callings from FusionCatcher
--output “/path/output_rna.txt”
</pre></div>
</div>
<ul class="simple">
<li><strong>SV_standard</strong> supports SV output from six callers (i.e.
STAR-fusion, Arriba, deFuse, FusionCatcher, SOAPFuse and Pizzly). For
some callers, several output files are generated in data processing
and the output format of different versions shows somewhat variable.
An extra note is made on which type of output from a caller (&gt;=
version X) is used as an input by SV_standard.<ul>
<li>For deFuse (&gt;=??)</li>
<li>For FusionCatcher (&gt;=??)</li>
<li>For Arriba (&gt;=??)</li>
<li>For STAR-Fusion (&gt;=??)</li>
<li>For SOAPFuse (&gt;=??)</li>
<li>For Pizzly (&gt;=??)</li>
</ul>
</li>
<li>file name for each sample in “/path/folder_Arriba”,
“/path/folder_STAR-Fusion” and “/path/FusionCatcher” should be
identical.</li>
</ul>
</div>
<div class="section" id="for-usage-of-sv-standard-arguments">
<h3>For usage of <strong>SV_standard</strong> arguments<a class="headerlink" href="#for-usage-of-sv-standard-arguments" title="Permalink to this headline">¶</a></h3>
<div class="highlight-default notranslate"><div class="highlight"><pre><span></span><span class="o">./</span><span class="n">SV_standard</span> <span class="o">--</span><span class="n">help</span>
<span class="o">--</span><span class="nb">type</span> <span class="p">{</span><span class="n">DNA</span><span class="p">,</span> <span class="n">RNA</span><span class="p">}</span>
<span class="c1">#// either DNA or RNA. If DNA is set, --Manta, --Delly, --Lumpy will be activated; otherwise --Arriba, --STAR-fusion, --deFuse, --SOAPFuse and –Pizzly are valid.</span>
<span class="o">--</span><span class="n">genome</span> <span class="p">{</span><span class="n">hg19</span><span class="p">,</span> <span class="n">hg38</span><span class="p">}</span>
<span class="c1">#// genome version</span>
<span class="o">--</span><span class="n">dis</span> <span class="p">{</span><span class="nb">max</span> <span class="n">distance</span> <span class="n">between</span> <span class="n">breakpoints</span> <span class="n">during</span> <span class="n">merging</span><span class="p">,</span> <span class="n">default</span><span class="p">:</span> <span class="mi">500</span><span class="p">}</span>
<span class="o">--</span><span class="n">sv_type</span> <span class="p">{</span><span class="n">whether</span> <span class="n">SV</span> <span class="nb">type</span> <span class="ow">is</span> <span class="n">needed</span> <span class="n">the</span> <span class="n">same</span> <span class="n">during</span> <span class="n">merging</span><span class="p">,</span> <span class="n">default</span><span class="p">:</span> <span class="n">yes</span><span class="p">}</span>
<span class="o">--</span><span class="n">sv_strand</span> <span class="p">{</span><span class="n">whether</span> <span class="n">SV</span> <span class="n">strand</span> <span class="ow">is</span> <span class="n">needed</span> <span class="n">the</span> <span class="n">same</span> <span class="n">during</span> <span class="n">merging</span><span class="p">,</span> <span class="n">default</span><span class="p">:</span> <span class="n">no</span><span class="p">}</span>
<span class="o">--</span><span class="n">size</span> <span class="p">{</span><span class="nb">min</span> <span class="n">size</span> <span class="n">of</span> <span class="n">SVs</span> <span class="n">to</span> <span class="n">be</span> <span class="n">taken</span> <span class="ow">in</span> <span class="n">account</span><span class="p">,</span> <span class="n">default</span><span class="p">:</span> <span class="mi">0</span><span class="p">}</span>
</pre></div>
</div>
</div>
<div class="section" id="run-sv-standard-via-singularity-container">
<h3>Run <strong>SV_standard</strong> via Singularity container<a class="headerlink" href="#run-sv-standard-via-singularity-container" title="Permalink to this headline">¶</a></h3>
<p>COMING SOON</p>
</div>
</div>
<div class="section" id="input-of-mutation-profile-optional">
<h2>Input of mutation profile (optional)<a class="headerlink" href="#input-of-mutation-profile-optional" title="Permalink to this headline">¶</a></h2>
<p>FuSViz only accepts input of mutation profile in Muation Annotation
Format (MAF), which is tab-delimited text file with aggregated mutation
information (SNVs+Indels) from multiple VCF files. In general, users can
use the tool <a class="reference external" href="https://github.com/mskcc/vcf2maf">vcf2maf</a> to convert
VCF to MAF and aggregate results of multiple samples.</p>
</div>
</div>


           </div>
           
          </div>
          <footer>
    <div class="rst-footer-buttons" role="navigation" aria-label="footer navigation">
        <a href="Usage_manual.html" class="btn btn-neutral float-right" title="Usage manual" accesskey="n" rel="next">Next <span class="fa fa-arrow-circle-right" aria-hidden="true"></span></a>
        <a href="Installation.html" class="btn btn-neutral float-left" title="Installation" accesskey="p" rel="prev"><span class="fa fa-arrow-circle-left" aria-hidden="true"></span> Previous</a>
    </div>

  <hr/>

  <div role="contentinfo">
    <p>
        &#169; Copyright 2021, Sen ZHAO.

    </p>
  </div>
    
    
    
    Built with <a href="https://www.sphinx-doc.org/">Sphinx</a> using a
    
    <a href="https://github.com/readthedocs/sphinx_rtd_theme">theme</a>
    
    provided by <a href="https://readthedocs.org">Read the Docs</a>. 

</footer>
        </div>
      </div>

    </section>

  </div>
  

  <script type="text/javascript">
      jQuery(function () {
          SphinxRtdTheme.Navigation.enable(true);
      });
  </script>

  
  
    
   

</body>
</html>